src/omit.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const omit = (obj, path) => {
+  if (obj === null) {
+    return {}
+  }
+  
+  const keys = Array.isArray(path) ? path : [path];
```

Comment:
I know lodash supports having single value here or spread params, but I don't think it seems good patter especially for pick to have one prop as it would simply better be obj[key] instead of whole function in that case